master/docs/manual/cfg-reporters.rst

```diff
@@ -879,6 +879,39 @@ As a result, we recommend you use https in your base_url rather than http.
     :param string password: the stash user's password
     :param list builders: only send update for specified builders
 
+.. bb:reporter::BitbucketStatusPush
+
+BitbucketStatusPush
+~~~~~~~~~~~~~~~~~~~
+
+.. @cindex BitbucketStatusPush
```

Comment:
we do not use this any more: I probably should go through our docs and remove all instances of texinfo legacy.